features/dd-sdk-android-ndk/src/androidTest/kotlin/com/datadog/android/ndk/NdkTests.kt

```diff
@@ -54,7 +54,7 @@ class NdkTests {
     @Test
     fun mustWriteAnErrorLog_whenHandlingSignal_whenConsentUpdatedToGranted() {
         val fakeSignal = forge.anInt(min = 1, max = 32)
-        // we need to keep this this size because we are using a buffer of [30] size in c++ for
+        // we need to keep this size because we are using a buffer of [30] size in c++ for
```

Comment:
> [!NOTE] 
> You're talking about the `(size = 20)` below right? Wouldn't it be better to have a constant `const val SIGNAL_SIZE = 20` (declared in the `companion object` for example) and reuse it everywhere?
> Or because we know the size can be at most 30, user something like `val signalSize = forge.anInt(1, 30)`?

Reply:
I actually need to check this but I think this is not needed anymore and I need to just forge a positive int there as the serialization was changed